src/DistortableImageOverlay.js

```diff
@@ -274,6 +274,21 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
     return this;
   },
 
+  getAngle: function() {
+    var matrix = this._image.style.transform
+      .split('matrix3d')[1]
```

Comment:
We want to add 2 extra spaces to each of these so that they're lined up with "matrix" and the linter doesn't complain:

```suggestion
        .split('matrix3d')[1]
```